packages/block-library/src/image/edit.native.js

```diff
@@ -242,6 +263,16 @@ class ImageEdit extends React.Component {
 					autoCorrect={ false }
 					keyboardType="url"
 				/>
+				{ // eslint-disable-next-line no-undef
+					__DEV__ &&
+					<BottomSheet.Cell
```

Comment:
Just curious:
Where you able to test `BottomSheet.PickerCell` component instead of the regular `Cell`?

Reply:
I did not test BottomSheet.PickerCell yet, but it looks like it would need to be adjusted to avoid the iOS issue where it freezes if an ActionSheet is shown before the initial Modal is dismissed. Right now it looks like it would try and show the picker right when the previous cell is clicked:  
https://github.com/WordPress/gutenberg/blob/5774046c216a96d95fcbcfd2903b74f5f69a0611/packages/components/src/mobile/bottom-sheet/picker-cell.native.js#L16-L25

I'll look into abstracting the show picker on dismiss functionality into PickerCell, since it seems this might be a good pattern to reuse elsewhere.

Reply:
See comment https://github.com/WordPress/gutenberg/pull/17245#issuecomment-528232485